### PR TITLE
Extra statically determined fields

### PR DESCRIPTION
**Pull request by muhammadahmadmslm**
- [x] Continent
- [x] Country Name
- [x] Currency
- [x] Is EU
- [x] Flag URL / Emoji 
- [x] Swiftlint
- [x] swiftformate

### CODE REVIEW

**Review by UmanShahzad — COMMENTED** [review]
Can you point me to an example of how the user will be using the library? Is it the test cases? And if so why is it parsing JSON in there?

File: README.md [hunk 1]
```diff
@@ -64,7 +65,79 @@ IPINFO.shared.getBatch(ipAddresses: ipAddresses, withFilter: false) { response,
     }
 }
 ```
-
+# Country Name Lookup
+This library provides a system to lookup country names through ISO2 country codes.
+```swift
+IPINFO.shared.lookupIP(ip: "8.8.8.8") { status, response, msg in
```
Comment (UmanShahzad) [inline README.md:71]: In your test cases I see `getDetail` and here `lookupIP`. Which is it?

**Review by UmanShahzad — CHANGES_REQUESTED** [review]
Have you run `swiftfmt` or similar on this codebase? Please run it on the entire thing in a separate PR and make a `.sh` script for running it properly in a `scripts/` folder.

File: Sources/ipinfoKit/Model/IPResponse/IPResponse.swift [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    
+    public var getAbuse: Abuse? {
+        return abuse
+    }
+    
+    public var getDomains: Domains? {
+        return domains
+    }
+    
+    public var description: String {
```
[the remark this answers is not in the document]
Comment (muhammadahmadmslm) [reply Sources/ipinfoKit/Model/IPResponse/IPResponse.swift]: Yes Brother, Swift provides a built-in way to get a string representation of an object using the String(describing:) initialiser. This initialiser creates a string representation of a value or instance.

```swift
let array = [1, 2, 3]
let stringFromArray = String(describing: array)

let dictionary = ["key": "value"]
let stringFromDictionary = String(describing: dictionary)

class MyClass {
    var property: String
    
    init(property: String) {
        self.property = property
    }
}

let myInstance = MyClass(property: "Hello, World!")

// Using String(describing:) to get the string representation
let stringRepresentation = String(describing: myInstance)

```

File: Tests/.DS_Store
Comment (UmanShahzad) [inline Tests/.DS_Store:1]: ?

**Review by UmanShahzad — COMMENTED** [review]
I think your swiftfmt has forced real `\t` tab characters. Don't think that's normal in Swift? Please fix!

**Review by UmanShahzad — APPROVED** [review]

**Pull request merged**